src/applications/journal_display/journal_printer.c

```diff
@@ -803,330 +804,330 @@ static void JournalDisplay_PrintPlayedAtBattleFacility(UnkStruct_ov81_021D1610 *
 
-    StringTemplate_Format(param0->template, param0->strbuf, v0);
-    Text_AddPrinterWithParamsAndColor(window, FONT_SYSTEM, param0->strbuf, 0, (16 * 4 + 16), TEXT_SPEED_NO_TRANSFER, TEXT_COLOR(1, 2, 0), NULL);
-    Strbuf_Free(v0);
+    StringTemplate_Format(journalManager->template, journalManager->strbuf, strbuf);
+    Text_AddPrinterWithParamsAndColor(window, FONT_SYSTEM, journalManager->strbuf, 0, (16 * 4 + 16), TEXT_SPEED_NO_TRANSFER, TEXT_COLOR(1, 2, 0), NULL);
+    Strbuf_Free(strbuf);
 }
 
-static void ov81_021D2A60(UnkStruct_ov81_021D1610 *param0, u16 *name, u8 param2, u8 idx)
+static void JournalPrinter_SetPlayerOrPokemonName(JournalManager *journalManager, u16 *name, u8 unused, u8 idx)
 {
-    Strbuf *v0 = Strbuf_Init(32, 42);
+    Strbuf *strbuf = Strbuf_Init(32, HEAP_ID_JOURNAL);
 
-    Strbuf_CopyChars(v0, name);
-    StringTemplate_SetStrbuf(param0->template, idx, v0, param2, 1, GAME_LANGUAGE);
-    Strbuf_Free(v0);
+    Strbuf_CopyChars(strbuf, name);
+    StringTemplate_SetStrbuf(journalManager->template, idx, strbuf, unused, TRUE, GAME_LANGUAGE);
+    Strbuf_Free(strbuf);
 }
 
-static void JournalDisplay_PrintSingleBattleEvent(UnkStruct_ov81_021D1610 *param0, Window *window, JournalEntryOnlineEvent *journalEntryOnlineEvent, u8 row)
+static void JournalPrinter_PrintSingleBattleEvent(JournalManager *journalManager, Window *window, JournalEntryOnlineEvent *journalEntryOnlineEvent, u8 row)
 {
-    Strbuf *v0;
+    Strbuf *strbuf;
 
     switch (journalEntryOnlineEvent->result) {
     case 0:
-        v0 = MessageLoader_GetNewStrbuf(param0->loader, journal_entries_beat_player_single);
+        strbuf = MessageLoader_GetNewStrbuf(journalManager->loader, journal_entries_beat_player_single);
         break;
     case 1:
-        v0 = MessageLoader_GetNewStrbuf(param0->loader, journal_entries_lost_to_player_single);
+        strbuf = MessageLoader_GetNewStrbuf(journalManager->loader, journal_entries_lost_to_player_single);
         break;
     case 2:
     default:
-        v0 = MessageLoader_GetNewStrbuf(param0->loader, journal_entries_tied_player_single);
+        strbuf = MessageLoader_GetNewStrbuf(journalManager->loader, journal_entries_tied_player_single);
         break;
     }
 
-    ov81_021D2A60(param0, journalEntryOnlineEvent->playerName1, journalEntryOnlineEvent->unused1, 0);
-    StringTemplate_Format(param0->template, param0->strbuf, v0);
-    Text_AddPrinterWithParamsAndColor(window, FONT_SYSTEM, param0->strbuf, 0, (((16 * 4 + 16) + 16) + 16) + row * 16, TEXT_SPEED_NO_TRANSFER, TEXT_COLOR(1, 2, 0), NULL);
-    Strbuf_Free(v0);
+    JournalPrinter_SetPlayerOrPokemonName(journalManager, journalEntryOnlineEvent->playerName1, journalEntryOnlineEvent->unused1, 0);
+    StringTemplate_Format(journalManager->template, journalManager->strbuf, strbuf);
+    Text_AddPrinterWithParamsAndColor(window, FONT_SYSTEM, journalManager->strbuf, 0, (((16 * 4 + 16) + 16) + 16) + row * 16, TEXT_SPEED_NO_TRANSFER, TEXT_COLOR(1, 2, 0), NULL);
```

Comment:
**suggestion**: This `(((16 * 4 + 16) + 16) + 16)` construct shows up a lot throughout the file. Do we know what it controls? If so, can we give it a constant so that it's easy to change?